website/pages/docs/drivers/external/containerd.mdx

```diff
@@ -234,6 +258,76 @@ before you can use `bridge` networks.
  $ sudo tar -C /opt/cni/bin -xzf cni-plugins.tgz
 ```
 
+Also, ensure your Linux operating system distribution has been configured
+to allow container traffic through the bridge network to be routed via iptables.
+These tunables can be set as follows:
+
+```hcl
+ $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-arptables
+ $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-ip6tables
+ $ echo 1 > /proc/sys/net/bridge/bridge-nf-call-iptables
+```
+
+To preserve these settings on startup of a nomad client node, add a file
+including the following to `/etc/sysctl.d/` or remove the file your Linux
+distribution puts in that directory.
+
+```hcl
+ net.bridge.bridge-nf-call-arptables = 1
+ net.bridge.bridge-nf-call-ip6tables = 1
+ net.bridge.bridge-nf-call-iptables = 1
+```
+
+## Port Forwarding
+
+nomad supports both `static` and `dynamic` port mapping.
+
+1. **Static ports**
+
+Static port mapping can be added in the `network` stanza.
+
+```hcl
+network {
+  mode = "bridge"
+  port "lb" {
+    static = 8889
+    to     = 8889
+  }
+}
+```
+Here, `host` port `8889` is mapped to `container` port `8889`.<br/>
+**NOTE:** static ports are usually not recommended, except for
+`system` or specialized jobs like load balancers.
+
+2. **Dynamic ports**
+
+Dynamic port mapping is also enabled in the `network` stanza.
+
+```hcl
+network {
+  mode = "bridge"
+  port "http" {
+    to = 8080
+  }
+}
+```
+Here, nomad will allocate a dynamic port on the `host` and that port
+will be mapped to `8080` in the container.
+
+You can also read more about `network stanza` in the [`nomad official documentation`](https://www.nomadproject.io/docs/job-specification/network).
```

Comment:
We don't need to put code blocks around `stanza`. Generally we don't want to inline links, but put them at the end of the page instead.

```suggestion
You can read more about configuring networking under the [`network`] stanza documentation.
```

Reply:
Fixed.